scripts/xtensa-build-all.sh

```diff
@@ -57,6 +57,25 @@ following environment variable:
 or:
      PRIVATE_KEY_OPTION='-DRIMAGE_PRIVATE_KEY=path/to/key'  $0 ...
 
+This script supports XtensaTools but only when installed in a specific
+directory structure, example:
+
+myXtensa/
+└── install/
+    ├── builds/
+    │   ├── RD-2012.5-linux/
+    │   │   └── Intel_HiFiEP/
+    │   └── RG-2017.8-linux/
+    │       ├── LX4_langwell_audio_17_8/
+    │       └── X4H3I16w2D48w3a_2017_8/
+    └── tools/
+        ├── RD-2012.5-linux/
+        │   └── XtensaTools/
+        └── RG-2017.8-linux/
+            └── XtensaTools/
+
+$ XTENSA_TOOLS_ROOT=/path/to/myXtensa $0 ...
+
```

Comment:
Does this mean we support these toolchain configs or is it just an example? Not sure we support RD-2012.5-linux anymore.

Reply:
@dbaluta I think the 2012 is for BYT config.

Reply:
I don't know whether Cadence still supports toolchains that old but if you search "2012.5" in this script you will see that it is still in use. Example: https://sof-ci.01.org/sofpr/PR3874/build8206/build/byt_xcc.txt

Anyway this is just an example so it doesn't really matter for this particular documentation PR.